optimism/src/keccak/column.rs

```diff
@@ -126,11 +126,11 @@ impl FoldingColumnTrait for Column {
 ///   155..2123 -> curr
 ///         155                                                                        2119
 ///          <--------------------------------if_round<---------------------------------->
-///          <-------------if_sponge------------->    
```

Comment:
@querolita Not sure which editor you're using, but it'd help to make it automatic...

https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

Reply:
(sorry if this was already discussed somewhere else, I realise this change already got into master before?)

Reply:
I am using VSCode, so that link should help!

Reply:
> did this change get into master before?)

This line was merged in this PR https://github.com/o1-labs/proof-systems/pull/1963, Misha approved so that's why you might not recall where it came from